### ISSUE TITLE
bug: Mobile accessibility

### ISSUE THREAD

**Issue by khairulhaaziq** [radix-vue/radix-vue#217]
Several issues I come across quickly when trying with mobile and safari
- [x] Dropdown wont close on click outside #9 
- [x] Select wont close on click outside #19 
- [x] Toggle(single) is hard to toggle, ToggleItem in ToggleGroup is fine
- [x] ToggleGroupItem focus styling issue

### Docs
- [ ] input border not showing(prob safari issue)

**Comment by zernonia** [comment]
That's some super valuable insight @sibbng ! Appreciate it!

**Comment by sibbng** [comment]
Can you try adding that line to where it is appropriate?

```ts
document.documentElement.addEventListener('click', () => {})
```

This should fix the issue outside click issue on Safari iOS.

**Comment by khairulhaaziq** [comment]
> Can you try adding that line to where it is appropriate?
> 
> ```ts
> document.documentElement.addEventListener('click', () => {})
> ```
> 
> This should fix the outside click issue on Safari iOS. If it works for you I can move this fix to our iOS workaround.

hi @sibbng thanks so much for this. This is a great help!. I'm not entirely sure what do you mean and where would we put it. Do you want to create a PR for the fix?

**Comment by sibbng** [comment]
Hi, I pushed a fix in VueUse. The fix is released as part of v10.3.0. Based on my tests, it fixes the outside click issues on iOS. Can you confirm that the outside click issues have been resolved for you as well?

**Comment by khairulhaaziq** [comment]
> Hi, I pushed a fix in VueUse. The fix is released as part of v10.3.0. Based on my tests, it fixes the outside click issues on iOS. Can you confirm that the outside click issues have been resolved for you as well?

Thanks so much! Fixed!
